apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -226,28 +222,44 @@ <h3 class="ledgerHead">
-                                                                <sh-select [options]="lc.flattenAccountList | async" [(ngModel)]="txn.particular" (selected)="selectAccount($event,txn)" [placeholder]="'Select Accounts'" [notFoundLink]="true" (noResultsClicked)="showQuickAccountModal()" [showClear]="false" [multiple]="false"
-                                                                    [ItemHeight]="67" [useInBuiltFilterForFlattenAc]="true">
+                                                            <td class="col-xs-2">
+
+                                                                <input class="form-control" [maxlength]="10"
+                                                                    autocomplete="off" name="bsdateledger" type="text"
+                                                                    [(ngModel)]="item.entryDate" bsDatepicker
+                                                                    [bsConfig]="{ dateInputFormat: giddhDateFormat }"
+                                                                    [outsideClick]="true">
+                                                            </td>
+                                                            <td class="col-xs-7 select2-parent select2-ledger">
+                                                                <sh-select [options]="lc.flattenAccountList | async"
+                                                                    [(ngModel)]="txn.particular"
+                                                                    (selected)="selectAccount($event,txn)"
+                                                                    [placeholder]="'Select Accounts'"
+                                                                    [notFoundLink]="true"
+                                                                    (noResultsClicked)="showQuickAccountModal()"
+                                                                    [showClear]="false" [multiple]="false"
+                                                                    [ItemHeight]="67"
+                                                                    [useInBuiltFilterForFlattenAc]="true">
                                                                     <ng-template #optionTemplate let-option="option">
                                                                         <ng-container *ngIf="!option.additional?.stock">
-                                                                            <a href="javascript:void(0)" class="list-item" style="border-bottom: 1px solid #ccc;">
+                                                                            <a href="javascript:void(0)"
+                                                                                class="list-item"
+                                                                                style="border-bottom: 1px solid #ccc;">
```

Comment:
avoid inline css

Reply:
ye phle ka code h ravindr ji mene revert kiya h vo lines uper niche hogyi

Reply:
Yes i know, can we fix this as improvement?